Documentation/build_qubes_packages.md

```diff
@@ -0,0 +1,320 @@
+
+* Navigate to `vmm-xen` sources.
+
+```bash
+$ cd ~/qubes/qubes-builder/qubes-src/vmm-xen
+```
+
+* Add your fork as new remote repository, pull code and checkout to your branch.
+
+```bash
+$ git remote add fork https://github.com/3mdeb/qubes-vmm-xen.git
+$ git pull fork
+remote: Enumerating objects: 21, done.
+remote: Counting objects: 100% (21/21), done.
+remote: Compressing objects: 100% (19/19), done.
+remote: Total 21 (delta 2), reused 21 (delta 2), pack-reused 0
+Unpacking objects: 100% (21/21), 36.53 KiB | 6.09 MiB/s, done.
+From https://github.com/3mdeb/qubes-vmm-xen
+ * [new branch]      inteltxt-support   -> fork/inteltxt-support
+ * [new branch]      main               -> fork/main
+ * [new branch]      release3.0         -> fork/release3.0
+ * [new branch]      xen-4.1            -> fork/xen-4.1
+ * [new branch]      xen-4.12           -> fork/xen-4.12
+ * [new branch]      xen-4.13           -> fork/xen-4.13
+ * [new branch]      xen-4.14           -> fork/xen-4.14
+ * [new branch]      xen-4.4            -> fork/xen-4.4
+ * [new branch]      xen-4.6            -> fork/xen-4.6
+ * [new branch]      xen-4.8            -> fork/xen-4.8
+ * [new branch]      xen-4.8-release3.2 -> fork/xen-4.8-release3.2
+You asked to pull from the remote 'fork', but did not specify
+a branch. Because this is not the default configured remote
+for your current branch, you must specify a branch on the command line.
+$ git checkout fork/inteltxt-support
+```
+
+* Navigate to `grub2` sources.
+
+```bash
+$ cd ~/qubes/qubes-builder/qubes-src/grub2
+```
+
+* Add your fork as new remote repository, pull code and checkout to your branch.
+
+```bash
+$ git remote add fork https://github.com/3mdeb/qubes-grub2.git
+$ git pull fork
+remote: Enumerating objects: 142, done.
+remote: Counting objects: 100% (86/86), done.
+remote: Compressing objects: 100% (45/45), done.
+remote: Total 142 (delta 40), reused 77 (delta 40), pack-reused 56
+Receiving objects: 100% (142/142), 6.39 MiB | 10.22 MiB/s, done.
+Resolving deltas: 100% (40/40), completed with 13 local objects.
+From https://github.com/3mdeb/qubes-grub2
+ * [new branch]      grub-2.06          -> fork/grub-2.06
+ * [new branch]      inteltxt-support   -> fork/inteltxt-support
+ * [new branch]      master             -> fork/master
+ * [new branch]      trenchboot_support -> fork/trenchboot_support
+You asked to pull from the remote 'fork', but did not specify
+a branch. Because this is not the default configured remote
+for your current branch, you must specify a branch on the command line.
+$ git checkout fork/inteltxt-support
+```
+
+* For GRUB we need to manually download grub-2.06.tar.xz package to provide
+  sources (other way would be to change builder.conf settings to allow using
+  our fork). To fetch manually run following command.
+
+```bash
+$ FETCH_CMD="curl --proto '=https' --proto-redir '=https' --tlsv1.2 --http1.1 -sSfL -o" make get-sources
```

Comment:
You can call `make get-sources-extra COMPONENTS="vmm-xen grub2"` from qubes-builder dir, to avoid specifying this manually.

Reply:
@TomaszAIR can you try it out?